scss/components/_case-body.scss

```diff
@@ -26,13 +26,15 @@
 }
 
 .civicase__case-details-panel--summary {
-  .civicase__case-body_tab.affix {
+  .civicase__case-body_tab.affix,
```

Comment:
Let's not use activity and case classes in the same file. 
What I suggest is to have a generic class `civicase__panel_tab` and add this class to the HTML which uses `civicase__activity-filter` and `civicase__case-body_tab`. 

Please let me know if it doesn't make sense.

Reply:
@swastikpareek As per our discussion, as this was a low priority comment and also because I honestly could not come up with a very solid reason to implement this, lets skip this now please.